Description: Temporarily disable rkyv
 Nothing in Debian currently depends on chrno+rkyv, so temporarily disable to
 allow upgrading rkyv to 0.8.
 .
 There is an attempt upstream: https://github.com/chronotope/chrono/pull/1619
Author: Blair Noctis <ncts@debian.org>
Forwarded: not-needed
Last-Update: 2025-01-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: chrono/Cargo.toml
===================================================================
--- chrono.orig/Cargo.toml
+++ chrono/Cargo.toml
@@ -67,23 +67,6 @@ default = [
 ]
 libc = []
 now = ["std"]
-rkyv = [
-    "dep:rkyv",
-    "rkyv/size_32",
-]
-rkyv-16 = [
-    "dep:rkyv",
-    "rkyv?/size_16",
-]
-rkyv-32 = [
-    "dep:rkyv",
-    "rkyv?/size_32",
-]
-rkyv-64 = [
-    "dep:rkyv",
-    "rkyv?/size_64",
-]
-rkyv-validation = ["rkyv?/validation"]
 std = ["alloc"]
 unstable-locales = ["pure-rust-locales"]
 wasmbind = [
@@ -120,7 +103,7 @@ default-features = false
 version = "0.8"
 optional = true
 
-[dependencies.rkyv]
+[disabled.dependencies.rkyv]
 version = "0.7.43"
 optional = true
 default-features = false
